src/components/forms/SearchBar/SearchSuggestionFormats.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { Col, Row } from '../../layout';
+import { Strong } from '../../typography';
+import {
+  SearchSuggestionDefaultPropType,
+  SearchSuggestionFilterPropType,
+} from './SearchSuggestions.types';
+
+const SearchSuggestionDefault = ({ suggestion }) => {
+  return suggestion.value;
+};
+
+const SearchSuggestionFilter = ({ suggestion }) => {
```

Comment:
We should rather export this as a reusable renderer which can be use in `suggestionRenderer` prop.

Reply:
changed to `renderSuggestionFilter`